docs/changes-in-v7.md

```diff
@@ -0,0 +1,225 @@
+- #1039 Added `vue/no-v-model-argument` rule that does not allow argument to v-model.
+- #1039 Added `vue/no-custom-modifiers-on-v-model` rule that reports when `v-model` is used with custom modifiers on Vue Component.
+
+#### Commons:
+
+- #1086 Added `vue/no-template-no-target-blank` rule that disallows using `target="_blank"` attribute without `rel="noopener noreferrer"` to avoid a security vulnerability.
+- #1114 Added `vue/no-unregistered-components` rule that disallow using components that are not registered inside templates.
+- #755 Added `vue/html-comment-indent` rule that enforce consistent indentation in HTML comments.
+- #755 Added `vue/html-comment-content-newline` rule that enforce unified line brake in HTML comments.
+- #755 Added `vue/html-comment-content-spacing` rule that enforce unified spacing in HTML comments.
+- #627 #1144 Added `vue/no-duplicate-attr-inheritance` rule that warn to apply `inheritAttrs: false` when it detects `v-bind="$attrs"` being used.
+- #871 #1145 Added `vue/no-unused-properties` rule that report unused properties.
+- #633 #1148 Added `vue/no-mutating-props` rule that reports mutation of component props.
+- #671 Added `vue/one-component-per-file` rule that checks if there is only one component per file.
+- #1072 Added `vue/no-potential-property-typo` rule that disallow a potential typo in your component options.
+- #1155 Added `vue/no-arrow-functions-in-watch` rule that disallow use an arrow function to define a watcher.
+- #1166 Added `vue/custom-event-name-casing` rule that enforces using kebab-case custom event names.
+- #1179 Added `vue/no-multiple-slot-args` rule disallows to pass multiple arguments to scoped slots.
+- #1185 Added `vue/no-bare-strings-in-template` rule that disallows the use of bare strings in `<template>`.
+- #1186 Added `vue/no-useless-v-bind` rule that reports `v-bind` with a string literal value.
+- #1187 Added `vue/no-useless-mustaches` rule that reports mustache interpolation with a string literal value.
+- #1191 Added `vue/no-restricted-v-bind` rule that disallow specific argument in `v-bind`.
+- #1192 Added `vue/no-restricted-static-attribute` rule that disallow specific attribute.
+- #1213 Added `vue/no-restricted-component-options` rule that disallow specific component options.
+- #1218 Added `vue/no-multiple-objects-in-class` rule disallows to pass multiple objects into array to class.
+- #1222 Added `vue/no-empty-component-block` rule disallows the `<template>` `<script>` `<style>` block to be empty.
+- #1238 Added `vue/no-lone-template` rule that disallow unnecessary `<template>` element.
+- #1239 Added `vue/no-dupe-v-else-if` rule that disallow duplicate conditions in `v-if` / `v-else-if` chains.
+- #1267 Added `v-for-delimiter-style` rule that enforces which delimiter (`in` or `of`) should be used in `v-for` directives.
```

Comment:
To stay consistent with the lines above:

```suggestion
- #1267 Added `vue/v-for-delimiter-style` rule that enforces which delimiter (`in` or `of`) should be used in `v-for` directives.
```

Reply:
Thank you!